libevmasm/Assembly.h

```diff
@@ -210,6 +226,11 @@ class Assembly
 
 	std::shared_ptr<std::string const> sharedSourceName(std::string const& _name) const;
 
+	void appendEOFHeader(bytes& bytecode, std::vector<size_t>& codeSectionSizeOffsets, size_t& startOfContainerSectionHeader, size_t& dataSectionSizeOffset) const;
```

Comment:
Parameters are missing the `_` prefix. it's also a bit long, so I'd wrap it:

```suggestion
	void appendEOFHeader(
		bytes& _bytecode,
		std::vector<size_t>& _codeSectionSizeOffsets,
		size_t& _startOfContainerSectionHeader,
		size_t& _dataSectionSizeOffset
	) const;
```

Reply:
Also, any reason not to make it `createEOFHeader()` and have it just return a tuple containing these values?